hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/TopologySubcommand.java

```diff
@@ -136,7 +144,8 @@ public Class<?> getParentType() {
   // Format
   // Location: rack1
   //  ipAddress(hostName) OperationalState
-  private void printOrderedByLocation(List<HddsProtos.Node> nodes) {
+  private void printOrderedByLocation(List<HddsProtos.Node> nodes,
+                                      String nState) throws IOException {
```

Comment:
```suggestion
                                      String state) throws IOException {
```

Reply:
The variable name 'state' was already there, so I have renamed it to nodeStateStr. Is that okay?

Reply:
I have fixed it now